### PR TITLE
add(sc): new stream-connector: PagerDuty

### PR DESCRIPTION
**Pull request by omercier**

### CODE REVIEW

File: en/integrations/stream-connectors/sc-pagerduty.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+### In PagerDuty
+
+There are two ways to integrate with PagerDuty: via Global Event Routing or on a PagerDuty Service.
+
+If you are adding this integration to an existing PagerDuty service, please skip to the Integrating with a PagerDuty Service section of this guide.
+
+#### Integrating With Global Event Routing
+
+Integrating with Global Event Routing enables you to route events to specific services based on the payload of the event from your tool. If you would like to learn more, please visit [our article on Global Event Routing](https://community.pagerduty.com/forum/t/service-configuration-guide/1660).
```
Comment (pkriko) [inline en/integrations/stream-connectors/sc-pagerduty.md:41]: Our article ?
This is pagerduty article right ? not our ?

File: en/integrations/stream-connectors/sc-pagerduty.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+![](https://pdpartner.s3.amazonaws.com/ig-template-incoming-event-source-key.png)
+
+#### Integrating With a PagerDuty Service
+
+1. From the **Configuration** menu, select **Services**.
+2. There are two ways to add an integration to a service:
+   * **If you are adding your integration to an existing service**: Click the **name** of the service you want to add the integration to. Then, select the **Integrations** tab and click the **New Integration** button.
+   * **If you are creating a new service for your integration**: Please read our documentation in section [Configuring Services and Integrations](https://support.pagerduty.com/docs/services-and-integrations#section-configuring-services-and-integrations) and follow the steps outlined in the [Create a New Service](https://support.pagerduty.com/docs/services-and-integrations#section-create-a-new-service) section, selecting **Centreon** as the **Integration Type** in step 4. Continue with the "[In Centreon](#in-centreon)" section once you have finished these steps.
```
Comment (pkriko) [inline en/integrations/stream-connectors/sc-pagerduty.md:53]: Read the pagerduty documentation ?

File: fr/integrations/stream-connectors/sc-pagerduty.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+| `pdy_centreon_url`  | String | `http://your.centreon.server`                    | URL du serveur Centreon                                                                                                |
+| `log_level`         | Number | 2 (valeur par défaut)                            | Niveau de verbosité des logs 0: errors seulement, 1: +warnings, 2: +verbose, 3: +debug                                 |
+| `log_path`          | String | `/var/log/centreon-broker/my-custom-logfile.log` | Chemin complet du fichier de log                                                                                       |
+| `max_buffer_size`   | Number | 1 (valeur par défaut)                            | Nombre maximum d'événements à stocker en mémoire tampon en attendant de les transmettre en un seul envoi               |
+| `max_buffer_age`    | Number | 5 (valeur par défaut)                            | Temps d'attente maximum avant d'envoyer les événements en mémoire tampon si `max_buffer_size` n'est pas encore atteint |
+
+**Remarks**
```
Comment (pkriko) [inline fr/integrations/stream-connectors/sc-pagerduty.md:129]: Remarques

File: en/integrations/stream-connectors/sc-pagerduty.md (hunk 1 above)
Comment (omercier) [reply en/integrations/stream-connectors/sc-pagerduty.md:41]: Yes, it was written as this in the template but it's theirs

**Pull request merged**